airbyte-webapp/src/components/ui/ResizablePanels/ResizablePanels.tsx

```diff
@@ -0,0 +1,107 @@
+import { faGripLinesVertical } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+import React from "react";
+import { ReflexContainer, ReflexElement, ReflexSplitter } from "react-reflex";
+import { useWindowSize } from "react-use";
+
+import styles from "./ResizablePanels.module.scss";
+
+interface ResizablePanelsProps {
+  leftPanel: {
+    children: React.ReactNode;
+    smallWidthHeader: React.ReactNode;
```

Comment:
Do these need to be react nodes, or could they just be strings?

Reply:
Changed the overlay header prop to a string 👍